python/py-tensorflow-tensorboard/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tensorflow-tensorboard
+version             0.4.0rc3
+platforms           darwin
+license             Apache-2
+maintainers         @emcrisostomo openmaintainer
+
+description         TensorFlow helps the tensors flow
+long_description    TensorBoard is a suite of web applications for inspecting \
+                    and understanding your TensorFlow runs and graphs.
+
+homepage            https://github.com/tensorflow/tensorboard
+master_sites        https://pypi.python.org/packages/d5/fb/80df4eb3234c41beeeb3122b1966effbf08608fb80d2fa74bb72f8da9cb3
+
+if {${python.version} < 30} {
+    distname            tensorflow_tensorboard-${version}-py2-none-any
+} else {
+    distname            tensorflow_tensorboard-${version}-py3-none-any
+}
+
+extract.suffix      .whl
+extract.cmd         cp
+extract.pre_args
+extract.post_args   ${worksrcpath}
+extract.mkdir       yes
+
+checksums           rmd160  52c4d0d3bf8856e2ebc87ae9785319e4416ccdce \
+                    sha256  2b8741aed01f35c95b781d25ebf1a84a45c2980746a616a36ade7ed8078d88ee
+
+python.versions     27 34 35 36
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+        port:py${python.version}-pip
+
+    depends_lib-append \
+        port:py${python.version}-wheel \
+        port:py${python.version}-bleach \
+        port:py${python.version}-markdown \
+        port:py${python.version}-html5lib \
+        port:py${python.version}-werkzeug \
+        port:py${python.version}-protobuf3 \
+        port:py${python.version}-six \
+        port:py${python.version}-numpy
+
+    if {${python.version} eq 27} {
+        depends_lib-append \
+            port:py${python.version}-futures \
+    }
+
+    build {
+    }
+
+    destroot.cmd        pip-${python.branch}
+    destroot.pre_args
```

Comment:
This would normally contain ${destroot.target}, which is `install` by default. So you could just leave that out of destroot.args instead.

Reply:
Thanks, I had no idea and it's cleaner.